zipline/pipeline/loaders/earnings_estimates.py

```diff
@@ -343,52 +515,51 @@ def create_overwrite_for_quarter(self,
         quarters_with_estimates_for_sid : np.array
             An array of all quarters for which there are estimates for the
             given sid.
+        requested_quarter : float
```

Comment:
I don't think it is necessary, was just surprised about the type. Thanks for explanation.